src/shared/CMakeLists.txt.inc

```diff
@@ -12,10 +12,11 @@ include_directories(${shared_dir}/vartypes/enhanced)
 include_directories(${shared_dir}/vartypes/gui)
 include_directories(${shared_dir}/vartypes/xml)
 
-include_directories("/usr/include/eigen2")
+# include_directories("/usr/include/eigen2")  # update for eigen3 at toplevel 01/16
```

Comment:
Yes, on some lesser devices (e.g. raspberry-pi ubuntu) there was some difficulty getting the eigen2 library, so the cmake dependency and single line of code were updated. (see 816042cf8d9c5f95f77a82c589c2b12c8772a62c)